aviatrix_ha.py

```diff
@@ -98,14 +98,19 @@ def _lambda_handler(event, context):
         restore_security_group_access(client, tmp_sg)
     try:
         instance_name = os.environ.get('AVIATRIX_TAG')
-        controller_instanceobj = client.describe_instances(
-            Filters=[
-                {'Name': 'instance-state-name', 'Values': ['running']},
-                {'Name': 'tag:Name', 'Values': [instance_name]}]
-        )['Reservations'][0]['Instances'][0]
+        inst_id = os.environ.get('INST_ID')
+        try:
+            controller_instanceobj = client.describe_instances(
+                Filters=[
+                    {'Name': 'instance-state-name', 'Values': ['running']},
+                    {'Name': 'tag:Name', 'Values': [instance_name]}]
+            )['Reservations'][0]['Instances'][0]
+        except IndexError:
+            controller_instanceobj = client.describe_instances(
```

Comment:
Could we add a log here? 

Also, we should not check if inst_id  is None (During creation)

Reply:
Sure, added a log, and only check if inst_id exists